manifests/README.cnv

```diff
@@ -0,0 +1,44 @@
+# CNV Manifests
+
+## KubeVirt
+
+Done as described here:
+
+https://kubevirt.io/user-guide/docs/latest/administration/intro.html#2-alternative-flow-aka-operator-flow
+
+Created using:
+
+```
+VERSION=v0.15.0
```

Comment:
This is prone to become stale. (Uness we bump it each new release.) Should we entertain fetching the manifests dynamically as part of ```make``` process?

Reply:
I do not plan to do this in this PR, this PR is about providing the initial stuff and seeing how it will be consumed.

but yes - anything we aim for on the long run needs to be automated.I'd even favor to get rid of this commited yamls in that case.

Reply:
The way this is handled in openshift-install is some code generates the yaml files, ref:

https://github.com/openshift/installer/tree/master/pkg/asset/manifests

And we have a fork of that repo for the kni work here:

https://github.com/metalkube/kni-installer/

So if we want to avoid the static yamls, they could be generated or fetched in that code?